File: fibonacci.java
public class Main {
    public static int fib(int number){ 
        if(number == 1 || number == 2){ 
            return 1; 
        } 
        return fib(number-1) + fib(number -2);
    }
    
    public static void fibonacci(int num) {
        for (int i = 1; i < num; i++) {
            System.out.println(fib(i));
        }
    }
    
    public static void main(String[] args) {
        fibonacci(10);
    }
}

File: fibonacci.kt
fun main(args: Array<String>) {
    fibonacci(10)
}

fun fibonacci(num: Int) {
    for (i in 1..num) {
        println(fib(i))
    }
}

fun fib(num: Int): Int {
    if (num == 1 || num == 2) {
        return 1
    }
    return fib(num-1) + fib(num-2)
}
